quickwit/quickwit-search/src/fetch_docs.rs

```diff
@@ -308,6 +308,7 @@ async fn create_snippet_generator(
     field: Field,
 ) -> anyhow::Result<SnippetGenerator> {
     let mut terms: Vec<&Term> = Vec::new();
+    // TODO ok with termset?
```

Comment:
what does the TODO mean?

Reply:
Ah is that working with TermSets?

well we should probably make sure the query_terms query is implemented and truncate the list
when there are too many terms.

Reply:
I don't know if that part of the code works properly with TermSetQuery, I haven't tested it yet. I don't know if it's important either, or can be considered a limitation, and queries with termset and snippet be rejected